build-java/action.yml

```
@@ -59,10 +63,10 @@ runs:
     - name: Set up Docker Buildx
       uses: docker/setup-buildx-action@v3
 
-    - name: Set up JDK 11
+    - name: Set up JDK
       uses: actions/setup-java@v4
       with:
-        java-version: '11'
+        java-version: '${{ inputs.javaVersion }}'
```

Think this would also work
```suggestion
        java-version: ${{ inputs.javaVersion }}
```

---

@aarontwf was thinking that too but left with the quotes just in case.